tests/internal/spatial-navigation-action-test.html

```diff
@@ -45,32 +45,26 @@
     let c3Button2 = document.querySelector('#c3b2');
 
     testRun(function() {
-      let prevScrollPosition;
+      let prevScrollPosition = 0;
       container1.scrollTop = 0;
       c1Button1.focus();
```

Comment:
I think more TC should be added.
There is not enough TC to verify the part you modified.
for example, focus should be entered into each container from outside or container element.
As I remember, it was bug case.